packages/headless/src/invoices.ts

```diff
@@ -15,6 +15,27 @@ import { MutationKey, QueryKey } from "@/constants";
 type OnSuccess = Void;
 type OnError = (error: Error) => void;
 
+type useFindInvoicesProps = {
+  query: UseInfiniteQueryResult<
+    InfiniteData<Paginated<IInvoice.Self>, unknown>,
+    Error
+  >;
+  list: IInvoice.Self[] | null;
+  more: () => void;
+};
+
+export function useFindInvoices({
+  pageParam,
+}: {
+  pageParam: number;
+}): useFindInvoicesProps {
+  const atlas = useAtlas();
+  const findInvoices = useCallback(async () => {
+    return await atlas.invoice.find({ page: pageParam });
+  }, []);
```

Comment:
This function should accept the `page` and the `size`. Similar to this one

https://github.com/litespace-org/litespace/blob/0795a725442639fd1ef03bb212a4304766f8577b/packages/headless/src/users.ts#L10-L13